src/IECoreGL/MeshPrimitive.cpp

```diff
@@ -99,11 +99,11 @@ void MeshPrimitive::addPrimitiveVariable( const std::string &name, const IECore:
 
 	if ( primVar.interpolation==IECore::PrimitiveVariable::FaceVarying )
 	{
-		addVertexAttribute( name, primVar.data );
+		addVertexAttribute( name, primVar.expandedData() );
```

Comment:
Given we're checking for constant primvars as indexed should we also check for indexed positions? Granted this might be unlikely but now the API supports it I think we should guard against it.

Reply:
Are you talking about in the code above, where we update the bounding box? Indexing is irrelevant there - we just want to visit each unique point once, which is what we're doing already.